frontend/src/app/libs/common-components/components/breadcrumb/breadcrumb.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="breadcrumb">
```

Comment:
It looks strange when **breadcrumb** component render **breadcrums**. 
My suggestion to name it maybe **breadcrumb-list** or **breadcrumb-panel**, smth like that.
Also, breadcrumb template can be placed in breadcrumb component (it's not necessary, can do it in separated task with refactoring).